source/includes/language-compatibility-table-php.rst

```diff
@@ -15,7 +15,7 @@
      - PHP 5.6
      - PHP 5.5
 
-   * - ext 1.9 + lib 1.8
```

Comment:
Just confirming that there is no `ext 1.9` and `lib 1.8` ?

Reply:
I don't understand the question. This PR is leaving "ext 1.9 + lib 1.8" as-is and only introducing a new row for "ext 1.10 + lib 1.9".

Reply:
I can resolve this. Initially the versions were updated and no new row added, making it appear that these versions were skipped